### DISCUSSION TITLE
Classification of alarm

### DISCUSSION THREAD

**Discussion by baiyibing123** [jertel/elastalert2#921] [Q&A] [answered]
Can I classify alarms by field?
For example, field A sends email to A and field B sends email to B

**Comment by jertel** [comment, marked as answer]
You can select the recipient of an alert via the `email_from_field`. See the [Email](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#email) documentation. If you need to use more than one field to change the recipient then you will either need to use multiple rules, or use an [enhancement](https://elastalert2.readthedocs.io/en/latest/recipes/adding_enhancements.html#enhancements).
